concordances/test_fixtures.go

```diff
@@ -0,0 +1,107 @@
+package concordances
+
+//This file contains common fixtures shared by all test files in this package
```

Comment:
Having this file is really bad practice, because this code will be included in the main build as the file name isn't _test.go.  This should be put into the test file it's most used (clients in service_test and service in handlers_test probably)